### ISSUE TITLE
Improve Accessibility

### ISSUE THREAD

**Issue by Tetrakern** [Tetrakern/fictioneer#4]
Accessibility has not been a priority in the development up to 5.0 because, frankly, I was running on fumes the whole time and this felt just like another burden on a pile I was already struggling with. So I ignored the most of it. Not all — I did not actively try to sabotage screen readers — but I cannot say I cared much.

The most simple issues are missing aria attributes, but I suspect the tab order is not particularly great either.
